conf/cut_release.sh

```diff
@@ -187,6 +192,20 @@ check_branch_dirty() {
     fi
 }
 
+check_npm_registry() {
+    if [[ ! $(npm config get registry) =~ (${YARN_PUBLIC_REGISTRY_REGEX}|${NPM_PUBLIC_REGISTRY_REGEX}) ]] ; then
+        printf "${red}Not pointing at the right npm registry! Make sure .npmrc points to ${NPM_PUBLIC_REGISTRY}${end}"
```

Comment:
`~/.npmrc`?

nit: One thing we could do is move the rc file aside while running the release, and set it back once complete (or if it aborts).

Reply:
Thanks I'll update ~/.npmrc.

I'd prefer not to modify the file system. I think with the `.npmrc` & `.yarnrc` in source control this should be a corner case. But if people start running into this we can look into making that change.

Reply:
As far as I know this yarn issue is still outstanding, meaning global rc files merge _on top_ of local: https://github.com/yarnpkg/yarn/issues/5371

Reply:
Did some testing and it seems to be merging correctly for me with yarn 1.13.0, giving priority to the working directory rc files

Reply:
I'm not sure it is, they claim it is still open and will be fixed in the next major version as a breaking change

https://github.com/yarnpkg/yarn/issues/5371#issuecomment-483274723